onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -754,6 +754,12 @@ def _where_input_wrangler(
         reason="fixme: ORT fails with invalid model: 'INVALID_ARGUMENT : Failed to load model with error: vector::_M_range_check: __n (which is 1) >= this->size() (which is 1)'",
         test_class_name="TestOutputConsistencyFullGraph",
     ),
+    xfail(
+        "nn.functional.cross_entropy",
+        reason="ORT < 1.15 fails on a few subtests with error 'index < data_.size() was false'. Resolved in ORT 1.15+",
```

Comment:
Should we bump ort version in test?

Reply:
To do that in a separate PR and clean up all similar decorators w.r.t ORT version?

Reply:
#770